src/main/java/frc/robot/sensors/VisionCamera.java

```diff
@@ -28,30 +26,21 @@ public VisionCamera(SerialPort jevois) {
         targetPoint = new Point(0, 0);
     }
 
-    public void updateVision() {
```

Comment:
If you delete the code don't check-in its commented remains below

Reply:
I needed to do this because we disabled the vision camera due to it making the robot periodically crash.

Reply:
The code is completely unusable in its current form however, due to the issue you referenced. If we need to re-insert this code when its fixed we can reference an old commit. If you need to disable code in a commit, don't just comment code, delete it.

Reply:
Alright, I'm adding it back in for dev's sake and for the 3-ball auto branch.